### PR TITLE
Add visual indicator to labels of required fields

### PR DESCRIPTION
**Pull request by Mister-42**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

**Comment by Mister-42** [comment]
Test on contact page: this change works, but there are empty labels on the page. These are now made visible by the presence of a red star.
`<label class="control-label control-label required" for="contactform-subject"></label>`

Investigating

File: src/Contact/ContactForm.php [hunk 1]
```diff
@@ -19,7 +19,7 @@ final class ContactForm extends FormModel
     public function attributeLabels(): array
     {
         return [
-            'username' => 'Username',
+            'username' => 'Name',
```
Comment (samdark) [inline src/Contact/ContactForm.php:22]:
```suggestion
            'name' => 'Name',
```

File: views/layout/main.php [hunk 2]
```diff
@@ -42,6 +42,12 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Yii Demo</title>
+    <style>
+        label.required:not(:empty):after {
```
Comment (samdark) [inline views/layout/main.php:46]: It's better to place it into `site.css`.

**Review by samdark — CHANGES_REQUESTED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**